lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # Licensed under the Apache2.0, see LICENCE file in charm source for details.
 """Library for the ingress relation.
```

Comment:
You may want to update the library to the latest version, we merged a large update in here https://github.com/canonical/nginx-ingress-integrator-operator/pull/47 Approving anyway

Reply:
There's another PR with the library :)